amplitude.py

```diff
@@ -64,7 +66,7 @@ def process_compressed (data):
         partitioned_events = partition_available_events(events)
         if is_partitioned(partitioned_events):
             events = partitioned_events[2]
-            batches = process(partitioned_events[0], is_last_call = False)
+            batches = process(pool, partitioned_events[0], batches, is_last_call = False)
```

Comment:
Unrelated, but ISTM we should be passing in the old value of `batches` here from previous iterations of the loop?